### DISCUSSION TITLE
Axios get method returns NaN in numeric field instead of value

### DISCUSSION THREAD

**Discussion by marcin-bury** [axios/axios#4981] [Q&A]
Hi

I have an Axiost GET request that that is supposed to return object from json. Among others the object contains numeric field "paymentCredit". The JSON returned from the server contains "paymentCredit":32 (checked in browser console), but Axios keeps returning NaN. It is quite strange since other numeric fields have proper values.

Any ideas where to look for the solution?

TIA
Marcin

**Comment by jasonsaayman** [comment]
nope this must be related to the api itself or how you implemented it
